cmd/cli/cmd/adi_test.go

```diff
@@ -35,14 +35,14 @@ func testCase2_1(t *testing.T, tc *testCmd) {
 	testCase5_1(t, tc)
 
 	//need to wait a sec to make sure faucet tx settles
-	time.Sleep(10 * time.Second)
+	time.Sleep(2 * time.Second)
```

Comment:
Gosh, this sure looks like this should be a constant rather than a hard coded value.